### PR TITLE
[v6r17] Split CPU and wallclock margin

### PR DESCRIPTION
**Pull request by fstagni**
BEGINRELEASENOTES

*WMS
CHANGE: split time left margins in cpuMargin and wallClockMargin. 

ENDRELEASENOTES

### CODE REVIEW

**Comment by fstagni** [comment]
Review anyone?

**Comment by phicharp** [comment]
I will tomorrow morning...

**Comment by fstagni** [comment]
OK I simplified. Review OK?

File: Core/Utilities/TimeLeft/TimeLeft.py [hunk 1]
```diff
@@ -215,4 +202,31 @@ def runCommand( cmd, timeout = 120 ):
...
 
 
+def enoughTimeLeft(cpu, cpuLimit, wallClock, wallClockLimit, cpuMargin, wallClockMargin):
+  """ Is there enough time?
+
+      :returns: True/False
+  """
+
+  cpuRemainingFraction = 1. - cpu / cpuLimit
+  wallClockUsedFraction = wallClock / wallClockLimit
```
Comment (phicharp) [inline Core/Utilities/TimeLeft/TimeLeft.py]: You can simplify as above as wallClockUsedFraction is not used
I would also multiply these 2 by 100 and removed 214,215 and the 100. * in 216
This is cosmetics but while there ;-)

Comment (fstagni) [inline Core/Utilities/TimeLeft/TimeLeft.py]: Done, please check it

**Comment by phicharp** [comment]
How did you manage without approval? ;-)
It was OK however!

**Pull request merged**